asplib_modules/core/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
-include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+#include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+
+# FIXME: Don't use globbing for source files
```

Comment:
I know that globbing for source files is very evil and I will change that for the final release. I only used that for developing because I changed the file structure and names very often. Consequently I didn't touch the CMakeLists.txt because a file name or folder was changed. Which was the reason that Core was core, because Windows doesn't handle this very well.